curvefs/src/mds/mds.cpp

```diff
@@ -163,6 +163,7 @@ void MDS::InitFsManagerOptions(FsManagerOption* fsManagerOption) {
            "default value: "
         << fsManagerOption->spaceReloadConcurrency;
 
+    fsManagerOption->mdsListenAddr = conf_->GetStringValue("mds.listen.addr");
```

Comment:
`mds.listen.addr` here is all mds addr, not the leader only

Reply:
it's the mds addr read from conf